src/averageAges.js

```diff
@@ -37,7 +42,21 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  // all women:
+  const women = people.filter(person => person.sex === 'f');
+  // all mothers:
+  const mothers = people
+    .map(person => person.mother)
+    .filter((mother, index, arrayOfMothers) =>
+      arrayOfMothers.indexOf(mother) === index && mother !== null);
```

Comment:
don't use `indexOf` here and try to avoid using the base array, you don't need it here